src/Sdk/DTWebApi/WebApi/ActionsStepTelemetry.cs

```diff
@@ -54,6 +61,9 @@ public ActionsStepTelemetry()
         [DataMember(EmitDefaultValue = false)]
         public TaskResult? Result { get; set; }
 
+        [DataMember(EmitDefaultValue = false)]
+        public TimelineRecordState? State { get; set; }
```

Comment:
why do we need the state? should it always be completed? since we have the result already.

Reply:
don't we have skipped etc.,?

Reply:
`TaskResult` should have all of them?

Reply:
state is always `completed` in that aspect I guess? Results need state, so trying to send that through

Reply:
since now I have a new class, set it just there